src/preset.rs

```diff
@@ -234,3 +234,161 @@ impl<I: IntoBindings> IntoBindings for Bidirectional<I> {
         positive.chain(negative)
     }
 }
+
+/// A preset to map buttons as 3-dimensional input.
+///
+/// Uses [`SwizzleAxis`] and [`Negate`] to bind inputs to the Y and Z directions.
+///
+/// See also [`Axial`] and [`Bidirectional`].
+///
+/// # Examples
+///
+/// Map keyboard inputs into a 3D movement action.
+///
+/// ```
+/// use bevy::prelude::*;
+/// use bevy_enhanced_input::prelude::*;
+///
+/// fn binding(
+///     trigger: Trigger<Binding<FlyCamera>>,
+///     settings: Res<KeyboardSettings>,
+///     mut cameras: Query<&mut Actions<FlyCamera>>,
+/// ) {
+///     let mut actions = cameras.get_mut(trigger.target()).unwrap();
+///     actions.bind::<Move>().to(SixDOF {
+///         forward: &settings.forward,
+///         right: &settings.right,
+///         backward: &settings.backward,
+///         left: &settings.left,
+///         up: &settings.up,
+///         down: &settings.down,
+///     });
+/// }
+///
+/// // We use `KeyCode` here because we are only interested in key presses.
+/// // But you can also use `Input` if you want to e.g.
+/// // combine mouse and keyboard input sources.
+/// #[derive(Resource)]
+/// struct KeyboardSettings {
+///     forward: Vec<KeyCode>,
+///     right: Vec<KeyCode>,
+///     backward: Vec<KeyCode>,
+///     left: Vec<KeyCode>,
+///     up: Vec<KeyCode>,
+///     down: Vec<KeyCode>,
+/// }
+///
+/// #[derive(InputContext)]
+/// struct FlyCamera;
+///
+/// #[derive(Debug, InputAction)]
+/// #[input_action(output = Vec3)]
+/// struct Move;
+/// ```
+#[derive(Debug, Clone, Copy)]
+pub struct SixDOF<I: IntoBindings> {
```

Comment:
What do you think about `Spatial` or `Tertial`?

Reply:
Spatial maybe, but I'm not sure it is better than SixDOF...

I guess it requires a bit less knowledge since you don't need to know DOF = Degrees of Freedom and what that means.

Reply:
@alice-i-cecile what do you think?

Reply:
If we decide to keep `SixDOF`, maybe worth to rename it to `SixDof` instead. The convention is to capitalize only the first letter in abbreviations - like `Aabb`.

Another option to consider: name these presets `<Something>1D`, `<Something>2D` and `<Something>3D`. Just brainstorming :slightly_smiling_face:

Reply:
Ya maybe `Axis1D`/`Axis2D`/`Axis3D`?

Reply:
I guess that might make axial vs axis2d confusing... Directional*D?

Reply:
I think I like "Spatial" best :) SixDoF is really confusing to new gamedevs.

Reply:
@Aceeri let's go with `Spatial` then!